packages/grid/_modules_/grid/components/panel/Panel.tsx

```diff
@@ -77,12 +49,13 @@ export function Panel(props: PanelProps) {
   return (
     <Popper
       placement="bottom-start"
-      open={open}
+      open={!!open}
```

Comment:
Why do we need the `!!` infront? I see that it is already marked as a `boolean`.

Reply:
Why not make the open state required?

Reply:
extra props need to be optional for customizable component

Reply:
What's the use case for using `<Panel open={false}>`?

Reply:
if you customize the panel you need to know if it's open or not

Reply:
So it should be required?

Reply:
yes but there are some typings constraints with the components slots implementation atm